### PR TITLE
fix: NearMobileWallet upgrades

### PR DESCRIPTION
**Pull request by GuillemGarciaDev**
# Description

Please include a summary of the wallet and how users can install it. Be sure to take a look at our guide on [Custom Wallets](https://github.com/near/wallet-selector/blob/main/packages/core/docs/guides/custom-wallets.md) to understand core concepts when integrating with Wallet Selector.

## Summary

This PR upgrades the improvements found by @kujtimprenkuSQA. Contains a `@peersyst/near-mobile-signer` package upgrade and a fix to the QR code.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "near-wallet-selector",
-  "version": "8.5.4",
+  "version": "8.5.5",
```
Comment (kujtimprenkuSQA) [inline package.json:3]: There's no need to bump the version on this PR, we will do it on the next release.

File: packages/near-mobile-wallet/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@near-wallet-selector/near-mobile-wallet",
-  "version": "8.5.3",
+  "version": "8.5.4",
```
Comment (kujtimprenkuSQA) [inline packages/near-mobile-wallet/package.json:3]: There's no need to bump the version on this PR, we will do it on the next release.

**Review by kujtimprenkuSQA — COMMENTED** [review]
Can you please update your branch with the latest changes on our `dev` branch?

**Comment by kujtimprenkuSQA** [comment]
Is it necessary to include these `devDepencencies` on NPM:

I see `parcel` included in dependencies, is it correct, should it be there since parcel is a bundler?

image

Looking at the Code Tab this is what has been pushed/published to NPM, the package.json in this location contains all the dependencies and devDependencies but in `/dist` there's a package.json file that does not have these devDependencies, can you double-check if this is what should be published to NPM.

The only concern is that these might not be really needed and are being installed on dApps for example take a look at this PR:
https://github.com/near/near-discovery/pull/777/files

image

**Comment by GuillemGarciaDev** [comment]
> Is it necessary to include these `devDepencencies` on NPM:
> 
> I see `parcel` included in dependencies, is it correct, should it be there since parcel is a bundler?
> 
> image
> 
> Looking at the Code Tab this is what has been pushed/published to NPM, the package.json in this location contains all the dependencies and devDependencies but in `/dist` there's a package.json file that does not have these devDependencies, can you double-check if this is what should be published to NPM.
> 
> The only concern is that these might not be really needed and are being installed on dApps for example take a look at this PR: https://github.com/near/near-discovery/pull/777/files
> 
> image

Hey @kujtimprenkuSQA!

Thank you for feedback. We removed the `devDependencies` in the `dist` directory to prevent them to be installed in `wallet-selector`. If you run `yarn` on the `wallet-selector` you will see that these devDependencies are not installed. About `parcel`, this bundler must be in our package because it helps us to test it.

Also, all requested changes are done!

**Comment by kujtimprenkuSQA** [comment]
> Hey @kujtimprenkuSQA!
> 
> Thank you for feedback. We removed the `devDependencies` in the `dist` directory to prevent them to be installed in `wallet-selector`. If you run `yarn` on the `wallet-selector` you will see that these devDependencies are not installed. About `parcel`, this bundler must be in our package because it helps us to test it.
> 
> Also, all requested changes are done!

I see, thanks for addressing devDependencies too.

**Review by kujtimprenkuSQA — APPROVED** [review]

**Pull request merged**